views/random-beer.hbs

```diff
@@ -0,0 +1,15 @@
+{{#beers}}
+{{>template this}}
+<p>
+<h1>Food Pairing</h1>
+    {{#each food_pairing}}
+    <ul>
+        <li>{{this}}</li>
+    </ul>
+
+    {{/each}}
+</p>
+<aside> <span>Brewers Tips :</span>
+    {{brewers_tips}}
+</aside>
+{{/beers}}
```

Comment:
If you pass the entire array, you could just use `#each` to set the sope. If you pass an element you could use `#with` to set the scope. Take a look here: [#with](https://handlebarsjs.com/guide/builtin-helpers.html#with)